pkg/internal/vault/vault.go

```diff
@@ -362,3 +363,17 @@ func (v *Vault) requestTokenWithKubernetesAuth(client Client, kubernetesAuth *v1
 func (v *Vault) Sys() *vault.Sys {
 	return v.client.Sys()
 }
+
+func (v *Vault) IsVaultInitializedAndUnsealed() error {
+	healthURL := path.Join("/v1", "sys", "health")
+	heatlhRequest := v.client.NewRequest("GET", healthURL)
```

Comment:
Spell err in this var

Reply:
`heatlhRequest` should be `healthRequest`

Reply:
Typo. My bad.